backends/platform/sdl/sdl.cpp

```diff
@@ -218,7 +218,9 @@ void OSystem_SDL::initBackend() {
 
 #if defined(USE_OPENGL_GAME) || defined(USE_OPENGL_SHADERS) || defined(USE_GLES2)
 	detectFramebufferSupport();
+#ifndef NINTENDO_SWITCH
```

Comment:
what is the reason not supporting AA detection?

Reply:
detectAntiAliasingSupport create multiple SDL window, which is currenlty not supported by Switch SDL2 implementation

Reply:
ok, does it support AA at all?

Reply:
It does, but you have to create an FBO for it and then resolve it into the backbuffer. The devkitpro OpenGL implementation does not support a multisample backbuffer.

Reply:
Tbh, I don't know. I'm currently asking to devkitpro devs.

Reply:
ok, then. please move "#ifndef NINTENDO_SWITCH" into "detectAntiAliasingSupport" function

Reply:
@usineur I'm pretty sure it does support it, you can query `GL_MAX_SAMPLES` to check. It just doesn't support a multisampled context which ScummVM expects, you can only do it by rendering to an FBO.

Reply:
Looking at the code we already have the necessary render-to-FBO code and ScummVM won't attempt to use a multisample context in that case. The only problem is it still uses a multisample context for detection.

We can very simply solve this however by adding an alternative implementation to the guard. We should simply query `GL_MAX_SAMPLES` and then run a loop `for (int i = 1; i <= maxSamples; i <<= 1)` which adds i to the list of sample sizes supported.

Reply:
indeed, it might work

Reply:
I don't see how to do that in "detectAntiAliasingSupport" function as glew hasn't be initialized yet, and a call to 
`glGetIntegerv(GL_MAX_SAMPLES, (GLint *)&maxSamples);` will result in a crash.

Reply:
In that case it would be necessary to move the call `detectAntiAliasingSupport`, which should be fine since the result of it is only used in the UI and nowhere else.